Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2255,9 +2262,26 @@ - (void)_startTextManipulationsWithConfiguration:(_WKTextManipulationConfigurati
     });
 }
 
-static WebCore::TextManipulationItemIdentifier coreTextManipulationItemIdentifierFromString(NSString *identifier)
+struct ItemIdentifiers {
+    WebCore::FrameIdentifier frameID;
+    WebCore::TextManipulationItemIdentifier itemID;
+};
+
+static std::optional<ItemIdentifiers> coreTextManipulationItemIdentifierFromString(NSString *identifier)
 {
-    return makeObjectIdentifier<WebCore::TextManipulationItemIdentifierType>(identifier.longLongValue);
+    auto identifiers = String(identifier).split('-');
```

Comment:
`StringView::split` is more efficient. So we can make a `String`, but then should `StringView::split` it.

Reply:
Fixed.